src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/SemanticsFactsService/AbstractSemanticFactsService.cs

```diff
@@ -30,7 +30,8 @@ internal abstract partial class AbstractSemanticFactsService : ISemanticFacts
             s.Kind == SymbolKind.Parameter ||
             s.Kind == SymbolKind.RangeVariable ||
             s.Kind == SymbolKind.Field ||
-            s.Kind == SymbolKind.Property;
+            s.Kind == SymbolKind.Property ||
+            (s.Kind == SymbolKind.NamedType && s.IsStatic);
```

Comment:
i don't think the IsStatic check is necessary.  do you need it?

Reply:
Yes, otherwise tests like ```TestNameVerbatimIdentifier2``` fail.

Reply:
i'm not really opposed to those failing :)  but we can go with the least disruptive approach here.